components/styles.js

```diff
@@ -76,16 +75,19 @@ const styles = StyleSheet.create({
     backgroundColor: 'transparent',
     width: 28,
     height: 28,
+    left: 13,
+    top: 13,
     borderRadius: 14,
+    position: 'absolute',
```

Comment:
Nope, it turns out we do need the position: 'absolute', since it's for dayCircleFiller, not eventIndicator. The calendar gets weird without it.